### PR TITLE
Format code using 'cargo fmt'

### PR DESCRIPTION
**Pull request by Atul9**

### CODE REVIEW

File: crox/src/main.rs [hunk 1]
```diff
@@ -62,19 +62,21 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
...
-                match event.timestamp_kind {
-                    TimestampKind::Start => EventType::Begin,
-                    TimestampKind::End => EventType::End,
-                    //Chrome does not seem to like how many QueryCacheHit events we generate
-                    TimestampKind::Instant => continue,
-                };
+            let event_type = match event.timestamp_kind {
+                TimestampKind::Start => EventType::Begin,
+                TimestampKind::End => EventType::End,
+                //Chrome does not seem to like how many QueryCacheHit events we generate
```
Comment (bjorn3) [inline crox/src/main.rs]:
```suggestion
                // Chrome does not seem to like how many QueryCacheHit events we generate
```

Comment (Atul9) [reply crox/src/main.rs]: done, I have updated the PR

**Comment by wesleywiser** [comment]
Are these changes made with the default rustfmt settings or did you tweak any of the defaults?

**Comment by Atul9** [comment]
These chanages are made by running cargo fmt on the master branch with default rustfmt settings. No, I didn't tweak any of the defaults.

**Pull request merged**